public/js/monitor.js

```diff
@@ -158,6 +158,9 @@ async function hashEmailAndSend(emailFormSubmitEvent) {
   for (const emailInput of emailForm.querySelectorAll("input[type=email]")) {
     emailForm.querySelector("input[name=emailHash]").value = await sha1(emailInput.value);
     emailInput.value = "";
+    emailInput.disabled = true;
+    emailInput.placeholder="";
```

Comment:
minorest of nits, but spaces around the `=` would make it easier to read (and more consistent).

Reply:
hahaha i'll fix it. 'minorest of nits' might be my new favorite phrase.